Feature/reformatting

rachelholladay opened the pull request:
Small Controller Edits + adding joint impedance controller + adding ntorque controller (still in the works)

justagist commented:
Hi! I see you have removed the PR, but you seem to have created many new files and made many changes! I'm glad to know this repo was useful to you. 
I have been considering including cartesian controllers (impedance and hybrid force-motion controllers, maybe) at some point, without introducing conflicts with the joint controllers. I think I could use your help with their implementations if you are interested. I have local implementations, but incorporating them into the repo would introduce some breaking changes that will have to be tested intensively again; so I have been procrastinating :p 
I haven't looked through your code yet (too many diffs :) ), but I will probably start with your implementation on cartesian control when I get a chance to work on this again. 
P.S. May I ask why you removed all moveit functionalities, and also what changes you made in your new joint impedance controller version?

rachelholladay commented:
Hi! I deleted the PR because it was accidental. I was intending to merge a branch into my forked master, not your main master. I definitely would not pull request on your branch a large number of fairly undocumented changes (especially without warning!). 

Your repo has been tremendously useful to me! It saved me an untold amount of time. Its definitely getting a big shout-out in my acknowledgement sections :) 

What I am working on is still in development and definitely requires a level of clean-up and documentation. So I wouldn't (if I were you) look at my fork until I've gotten to that. I too have been procrastinating slightly on this front, and would definitely be interested in helping out and merging at some point.  A hybrid force-motion controller is on my list (but slightly on the back burner, since its won't be used for the paper I'm  currently trying to right). 

Re: MoveIt. I am not using MoveIt, so I removed it to save myself the trouble of having to install it.
